1.8/administration/access-node/tunnel.md

```diff
@@ -110,20 +110,38 @@ DC/OS Tunnel provides you with full access to the DNS, masters, and agents from
 
 ## Prerequisites
 * The [DC/OS CLI](/docs/1.8/usage/cli/install/).
-* The DC/OS Tunnel package. Run `dcos package install tunnel-cli --cli` from the DC/OS CLI.
+* The DC/OS Tunnel package. Run `dcos package install tunnel-cli --cli`.
 * [SSH access](/docs/1.8/administration/access-node/sshcluster/) (key authentication only).
 * [The OpenVPN client](https://openvpn.net/index.php/open-source/downloads.html) for VPN functionality.
```

Comment:
Are you referring to this line?
```
## VPN
 Run the following command from the DC/OS CLI
```

I think it fits better as a prerequisite, as it's required for more than just the VPN.